### PR TITLE
Alex dev ops

### PR DESCRIPTION
**Pull request by asp78**
Makes changes for allowing for compile-code and changes clean-hard.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -10,6 +10,7 @@ clean:
 
 clean-hard: clean
 	@ cd ../LocalDB; make clean -s
+	@ rm -rf ./logs/
```
Comment (smc395) [inline makefile]: Are we gonna keep the logs later for production stuff?

File: Frontend/package.json [hunk 2]
```diff
@@ -24,6 +24,10 @@
     "@angular/platform-browser": "~2.4.0",
     "@angular/platform-browser-dynamic": "~2.4.0",
     "@angular/router": "~3.4.0",
+    "@types/core-js": "^0.9.34",
+    "@types/jasmine": "^2.5.36",
+    "@types/node": "^6.0.45",
+    "@types/selenium-webdriver": "^2.53.33",
```
Comment (smc395) [inline Frontend/package.json]: Why was this moved? Testing should be a dev thing not prod right?

File: makefile (hunk 1 above)
Comment (asp78) [inline makefile]: Yes, this never is called in production code

File: Frontend/package.json (hunk 2 above)
Comment (asp78) [inline Frontend/package.json]: They are moved because without them the code will not compile with tsc.
